configs/prod/roles/inventory.json

```diff
@@ -44,11 +44,12 @@
     },
     {
       "name": "Inventory Groups Administrator",
```

Comment:
Unfortunately, the name itself can't be updated. Cause it is unique identifier, updating it would delete the original one and create a new role, instead of updating it in-place